contracts/profit/src/state/buy_back.rs

```diff
@@ -157,3 +176,29 @@ where
 }
 
 impl<I> TryFind for I where I: Iterator + ?Sized {}
+
+#[cfg(debug_assertions)]
+#[derive(Debug)]
+struct BlankVisitor;
+
+#[cfg(debug_assertions)]
+mod blank_visitor_impl {
+    use serde::{de::DeserializeOwned, Serialize};
+
+    use finance::currency::{AnyVisitor, AnyVisitorResult, Currency};
+    use platform::never::Never;
+
+    use super::BlankVisitor;
+
+    impl AnyVisitor for BlankVisitor {
+        type Output = ();
+        type Error = Never;
+
+        fn on<C>(self) -> AnyVisitorResult<Self>
+        where
+            C: Currency + Serialize + DeserializeOwned,
+        {
+            Ok(())
+        }
+    }
+}
```

Comment:
it looks like an overkill solution ... and non-applicable since we do not have Payment-NLS group